tornadoes/__init__.py

```diff
@@ -10,62 +10,66 @@
 
 
 class ESConnection(object):
+    _MATCH_ALL_QUERY = {"query": {"match_all": {}}}
 
-    def __init__(self, host='localhost', port='9200', io_loop=None, protocol='http', custom_client=None):
+    # TODO : timeout, max_retries, retry_on_timeout
+    def __init__(self, host='localhost', port='9200', io_loop=None, protocol='http', custom_client=None,
+                 http_request_kwargs=None):
         self.io_loop = io_loop or IOLoop.instance()
         self.url = "%(protocol)s://%(host)s:%(port)s" % {"protocol": protocol, "host": host, "port": port}
         self.bulk = BulkList()
         self.client = custom_client or AsyncHTTPClient(self.io_loop)
-        self.httprequest_kwargs = {}     # extra kwargs passed to tornado's HTTPRequest class e.g. request_timeout
+
+        # extra kwargs passed to tornado's HTTPRequest class e.g. request_timeout
+        self.http_request_kwargs = http_request_kwargs or {}
 
     @staticmethod
-    def from_uri(uri, io_loop=None, custom_client=None):
+    def _create_query_string(params):
```

Comment:
I always write private methods before public methods. I know what you think about the order of method but I don't agree with you.